fiboa_cli/datasets/dk.py

```diff
@@ -0,0 +1,163 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+import re
+import pandas as pd
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# data original source : https://landbrugsgeodata.fvm.dk/
+URI = "/mnt/c/kerner-lab/fieldscapes/denmark/all_parcels.gpkg" # data seubset for FieldScapes
```

Comment:
Same as in #43: I'm not sure how to proceed with this. It's not really useful for the general public if there's no way to get the source data. Any thoughts from the fieldscapes members?

Reply:
The core idea here is to get the converter code out asap, but the first goal is to get the resulting files into fiboa on source cooperative. The hope is that @eddiechoi00 will be able to help with adapting the converters to work with the original source data, and then putting the full datasets (instead of the subsets fieldscapes is focused on) in source cooperative.

Reply:
Yeah, I was thinking the same. We should make it so that the source data can get converted and then there could be an option that people can enable to run on fieldscapes data.